package.json

```diff
@@ -20,5 +20,9 @@
   "dependencies": {
     "breeze-dag": "^0.1.0",
     "through2": "^2.0.0"
+  },
+  "devDependencies": {
```

Comment:
Shouldn't these two be peerDependencies instead, because they are installed by the aurelia skeleton?